target/xilinx/scripts/util/flash.tcl

```diff
@@ -39,6 +39,22 @@ switch $board {
             PROGRAM.CHECKSUM {0} \
             ] $hw_cfgmem
     }
+    vcu118 {
```

Comment:
Again, no need to duplicate code, just add another case to the block above (add `vcu128 -` over it)

I realize now that the TODO about Genesys2 above is a leftover; Genesys2 flashing is also supported with the existing block. We can remove it.

Reply:
Sure!